mkosi/__init__.py

```diff
@@ -2899,8 +2962,13 @@ def install_boot_loader(args: CommandLineArguments, root: str, loopdev: Optional
         return
 
     with complete_step("Installing boot loader"):
-        if args.esp_partno and args.distribution != Distribution.clear:
-            run_workspace_command(args, root, ["bootctl", "install"])
+        if args.esp_partno:
+            if args.distribution == Distribution.clear:
+                pass
+            elif args.distribution in (Distribution.centos, Distribution.centos_epel) and is_older_than_centos8(args.release):
```

Comment:
This line was broken above and should be broken here as well.

Reply:
Please check if I understood your remark correctly, I was a bit confused by the wording. :confused: